src/main/java/com/newts/newtapp/api/application/user/Block.java

```diff
@@ -31,7 +31,7 @@ public Void request(RequestModel request) throws UserNotFound, UserAlreadyBlocke
 
```

Comment:
this method should also make the users unfollow eachother if they currently are.

Reply:
I believe the block interactor already does that right?

Reply:
yes you're right